### DISCUSSION TITLE
Why wire:navigate does not work in livewire app?

### DISCUSSION THREAD

**Discussion by sergeynilov** [livewire/livewire#8822] [1. Help] [answered]
I want to use SPA in my laravel 11 / livewire 3.5 app and I failed to navigate as it SPA - as I see all pages are reloaded.
in routes/web.php I have :

    Route::get('/', HomeContainer::Class)->name('home');
    Route::get('/partnership-order', PartnershipOrderContainer::class)->name('partnership_order.index');

Firstly I have Some navigator component, but as SPA does not work I simplified it as in app/Livewire/PartnershipOrderContainer.php :

    <?php
    
    namespace App\Livewire;
    
    use Livewire\Component;
    
    class PartnershipOrderContainer extends Component
    {
        public function render()
        {
            return view('livewire.partnership-order-container')->layout('layouts.frontend');
        }
    }

and in resources/views/livewire/partnership-order-container.blade.php:

    <div>
        resources/views/livewire/partnership-order-container.blade.php
        <livewire:counter>
    
        <a href="{{  route('home') }}" wire:navigate
           class="block py-2 px-3 text-white bg-blue-700 rounded md:bg-transparent md:text-blue-700 md:p-0 md:dark:text-blue-500  capitalize-text"
           aria-current="page"> 11 {{ __('home.home') }}</a>
    
    </div>

I left counter om blae to check that livewire works ok.

The second component app/Livewire/Home/HomeContainer.php:

    namespace App\Livewire\Home;
    
    use Livewire\Component;
    
    class HomeContainer extends Component
    {
        public function render()
        {
            return view('livewire.home.home-container')->layout('layouts.frontend');
        }
    }

and in resources/views/livewire/home/home-container.blade.php :

    <div>
     resources/views/livewire/home/home-container.blade.php
    
        <livewire:counter>
    
    
        <a href="{{  route('partnership_order.index') }}"
           class="block py-2 px-3 text-gray-900 rounded hover:bg-gray-100 md:hover:bg-transparent md:hover:text-blue-700 md:p-0 dark:text-white md:dark:hover:text-blue-500 dark:hover:bg-gray-700 dark:hover:text-white md:dark:hover:bg-transparent dark:border-gray-700  capitalize-text">22  {{ __('home.partnership_order') }} </a>
    
    </div>

  In used template resources/views/layouts/frontend.blade.php :

    <html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
    <head>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1">
        <meta name="csrf-token" content="{{ csrf_token() }}">
    
        <title>{{ config('app.name', 'Laravel') }}</title>
    
        
        <link rel="stylesheet" href="https://fonts.bunny.net/css2?family=Nunito:wght@400;600;700&display=swap">
    
        
        @vite(['resources/css/app.css', 'resources/js/app.js'])
    
        <wireui:scripts />
    
        @livewireStyles
    
    </head>
    <body class="font-sans antialiased">
    <div class="min-h-screen bg-gray-100 d2">
        <main>
             {{ $slot }}
        </main>
    </div>
      @livewireScripts
    
    @livewire('wire-elements-modal')
    </body>
    </html>

and config file config/livewire.php has options :

    /*
    |---------------------------------------------------------------------------
    | Auto-inject Frontend Assets
    |---------------------------------------------------------------------------
    |
    | By default, Livewire automatically injects its JavaScript and CSS into the
    | <head> and <body> of pages containing Livewire components. By disabling
    | this behavior, you need to use @livewireStyles and @livewireScripts.
    |
    */
    
    'inject_assets' => true,
    
    /*
    |---------------------------------------------------------------------------
    | Navigate (SPA mode)
    |---------------------------------------------------------------------------
    |
    | By adding `wire:navigate` to links in your Livewire application, Livewire
    | will prevent the default link handling and instead request those pages
    | via AJAX, creating an SPA-like effect. Configure this behavior here.
    |
    */
    
    'navigate' => [
        'show_progress_bar' => true,
        'progress_bar_color' => '#2299dd',
    ],

But making redirecting from one page to another I do not see SPA working properly :

https://www.youtube.com/watch?v=8KvHBi4P-dM

How do you see if SPA DOES NOT work properly and what is wrong with it ?

**Comment by EdgarsJoja** [comment, marked as answer]
your "22..." link is missing `wire:navigate` attribute...
Link "11..." has it, and seems to be navigating in SPA-like mode just fine.
